pages/instructorAssessment/instructorAssessment.sql

```diff
@@ -18,109 +81,47 @@ FROM
-    GROUP BY
-        user_id,
-        question_id,
-        aq.id
-),
-mean_assessment_scores AS (
-    SELECT
-        ai.user_id,
-        avg(ai.score_perc) AS user_score_perc,
-        ntile(5) OVER () as quintile
-    FROM
-        assessment_instances AS ai
-    WHERE
-        ai.assessment_id = $assessment_id
-    GROUP BY
-        user_id
-),
-question_users AS (
-    SELECT
-        aq.question_id,
-        ai.user_id
-    FROM
-        instance_questions AS iq
-        JOIN assessment_instances AS ai ON (iq.assessment_instance_id=ai.id)
-        JOIN assessment_questions AS aq ON (iq.assessment_question_id=aq.id)
-    WHERE
-        aq.assessment_id = $assessment_id AND
-        aq.deleted_at IS NULL
-    GROUP BY
-        user_id,
-        question_id,
-        aq.id
-),
-assessment_score_quintiles AS (
-    SELECT
-        mean_assessment_scores.user_id,
-        ntile(5) OVER (ORDER BY mean_assessment_scores.user_score_perc) AS quintile
-    FROM
-        mean_assessment_scores
-),
-question_score_quintiles AS (
-    SELECT
-        question_users.question_id AS question_id,
-        assessment_score_quintiles.quintile AS user_quintile,
-        avg(mean_question_scores.user_score_perc) AS score_perc
-    FROM
-        question_users
-        JOIN assessment_score_quintiles ON (assessment_score_quintiles.user_id = question_users.user_id)
-        JOIN mean_question_scores ON (mean_question_scores.question_id = question_users.question_id AND mean_question_scores.user_id = question_users.user_id)
-    GROUP BY
-        assessment_score_quintiles.quintile,
-        question_users.question_id
-    ORDER BY
-        assessment_score_quintiles.quintile
-),
-question_score_quintiles_condensed AS (
-    SELECT
-        question_score_quintiles.question_id AS question_id,
-        array_agg(question_score_quintiles.score_perc) AS quintile_scores
-    FROM question_score_quintiles
-    GROUP BY
-        question_score_quintiles.question_id
-)
 SELECT
     q.title,
-    avg(mean_question_scores.average_number_attempts) as average_number_attempts,
-    mean_question_scores.question_id,
-    greatest(0, least(100, avg(mean_question_scores.user_score_perc))) as mean_score_per_question,
-    mean_question_scores.number,
-    greatest(0, least(100, corr(mean_question_scores.user_score_perc, mean_assessment_scores.user_score_perc) * 100.0)) as discrimination,
-    question_score_quintiles_condensed.quintile_scores
-FROM mean_question_scores
-    JOIN mean_assessment_scores ON (mean_assessment_scores.user_id = mean_question_scores.user_id)
-    JOIN questions AS q ON (q.id = mean_question_scores.question_id)
-    JOIN question_score_quintiles_condensed ON (mean_question_scores.question_id = question_score_quintiles_condensed.question_id)
+    q.id AS question_id,
+    aq.id AS assessment_question_id,
+    admin_assessment_question_number(aq.id) as number,
+    aq.mean_score AS mean_score_per_question,
+    aq.discrimination,
```

Comment:
Decided to just delete this whole section and use statistics from the 'questions' block instead.